modules/installation-aws-user-infra-installation.adoc

```diff
@@ -8,7 +8,7 @@
 
 After you start the {product-title} installation on Amazon Web Service (AWS)
 user-provisioned infrastructure, remove the bootstrap node, reconcile the default
-Machine and MachineSet definitions, and delete unused nodes
+Machine and MachineSet definitions, and delete unused nodes:
```

Comment:
nit: Trailing colon seems odd to me, since the next thing is the prereq section.  Maybe use a trailing period instead?